docs/tutorials/seqcopy.md

```diff
@@ -1,4 +1,5 @@
 # Sequence copy model
+
```

Comment:
The sequence copy tutorial previously assumed you were in the tutorials folder where the script lies. Now that we're hosting the tutorial separately, this assumption no longer makes sense sense. Would it make sense to add something like
```
wget https://raw.githubusercontent.com/awslabs/sockeye/master/tutorials/seqcopy/genseqcopy.py
```

to the start of the tutorial itself?